### ISSUE TITLE
Accessibility: indicate which thumbnail is selected, for screenreaders

### ISSUE THREAD

**Issue by lhenze** [ProjectMirador/mirador#3907]
This issue is a subticket of https://github.com/ProjectMirador/mirador/issues/3578
It concerns the last bullet point, 
Thumbnail panel at the bottom lack aria-selected.

See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

Since the element in question has role `gridcell` , I agree that adding aria-selected to the selected thumbnail makes sense.  

So for each `div` in the thumbnail area where `role` is `gridcell` , let's add the   `aria-selected` attribute, set to `true` or `false` as appropriate.

**Comment by brittnylapierre** [comment]
I think wee determined this was related to this ticket https://github.com/ProjectMirador/mirador/issues/3578 

Which is related to a PR ready for review :D https://github.com/ProjectMirador/mirador/pull/3909
